kord-extensions/src/main/resources/translations/kordex/strings.properties

```diff
@@ -65,6 +65,26 @@ checks.topRoleHigher.failed=Must have a top role higher than: **{0}**
 checks.topRoleLower.failed=Must have a top role lower than: **{0}**
 checks.topRoleHigherOrEqual.failed=Must have a top role of **{0}**, or a higher top role
 checks.topRoleLowerOrEqual.failed=Must have a top role of **{0}**, or a lower top role
+cooldown.notifier.commandUser=You may use `{1}` again {0}.
+cooldown.notifier.commandUserChannel=You may use `{1}` in {2} again {0}.
+cooldown.notifier.commandUserGuild=You may use `{1}` in this server again {0}.
+cooldown.notifier.globalUser=You may use commands again {0}.
+cooldown.notifier.globalUserChannel=You may use commands in {1} again {0}.
+cooldown.notifier.globalUserGuild=You may use commands in this server again {0}.
+cooldown.notifier.global=Someone may use commands again {0}.
+cooldown.notifier.globalChannel=Someone may use commands in {1} again {0}.
+cooldown.notifier.globalGuild=Someone may use commands in this server again {0}.
+cooldown.notifier.generic=You've hit a cooldown of type `{0}`.
+ratelimit.notifier.commandUser=You are being rate limited from using `{1}` until {0}.
+ratelimit.notifier.commandUserChannel=You are being rate limited from using `{1}` in {2} until {0}.
+ratelimit.notifier.commandUserGuild=You are being rate limited from using `{1}` in this server until {0}.
+ratelimit.notifier.globalUser=You are being rate limited from using commands until {0}.
+ratelimit.notifier.globalUserChannel=You are being rate limited from using commands in {1} until {0}.
+ratelimit.notifier.globalUserGuild=You are being rate limited from using commands in this server until {0}.
+ratelimit.notifier.global=Commands cannot be used until {0}.
+ratelimit.notifier.globalChannel=Commands cannot be used in {1} until {0}.
+ratelimit.notifier.globalGuild=Commands cannot be used in this server until {0}.
+ratelimit.notifier.generic=You have hit a rate limit of type: `{0}`.
```

Comment:
A lot of these strings are far too verbose, users should not be told what types of limits they're hitting - this is an implementation detail that shouldn't be exposed to users, especially as you don't want them trying to work around the limits.

Don't tell them when the limit ends, either, 

It's important that these be kept generic, for that reason. Do not expose implementation details for bot security features, ever.

It's OK to keep separate strings that are capable of displaying this data, just in case bot developers do want to expose it in specific situations, but we cannot display it by default.